### PR TITLE
README: fix flags, links, and project description

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,14 @@
 [![Discord Server](https://img.shields.io/discord/844600078504951838?color=7289da&logo=discord "Discord Server")](https://discord.gg/jTysUaxXzM)
 [![Twitter](https://img.shields.io/twitter/follow/authzed?color=%23179CF0&logo=twitter&style=flat-square "@authzed on Twitter")](https://twitter.com/authzed)
 
-SpiceDB is a [Zanzibar]-inspired database that stores, computes, and validates application permissions.
+SpiceDB is a database system for managing security-critical permissions checking.
```
Comment (ecordell) [inline README.md:10]: "Managing ... checking" reads oddly to me?

**Review by ecordell — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jzelinskie) [reply README.md:10]: >SpiceDB is a database system for managing security-critical application permissions.

how about this?

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ See [CONTRIBUTING.md] for instructions on how to contribute and perform common t
 [gRPC]: https://buf.build/authzed/api
 [HTTP]: https://petstore.swagger.io/?url=https://raw.githubusercontent.com/authzed/authzed-go/main/proto/apidocs.swagger.json
 [Zanzibar]: https://authzed.com/blog/what-is-zanzibar/
```
Comment (josephschorr) [inline README.md:28]: Link not used after change above

**Review by ecordell — APPROVED** [review]
LGTM

**Pull request merged**